packages/cli/src/commands/doctor/runAutomaticFix.js

```diff
@@ -0,0 +1,47 @@
+import chalk from 'chalk';
+import ora from 'ora';
+import logger from '../../tools/logger';
+
+const AUTOMATIC_FIX_LEVELS = {
+  ALL_ISSUES: 'ALL_ISSUES',
+  ERRORS: 'ERRORS',
+  WARNINGS: 'WARNINGS',
+};
+
+const runAutomaticFix = async ({issues, automaticFixLevel, stats, loader}) => {
+  // Remove the fix options from screen
+  process.stdout.moveCursor(0, -6);
+  process.stdout.clearScreenDown();
+
+  // TODO: check `automaticFixLevel`, this should match `AUTOMATIC_FIX_LEVELS.ALL_ISSUES`
+
+  const totalIssues = stats.errors + stats.warnings;
+
+  logger.log(
+    `\nAttempting to fix ${chalk.bold(totalIssues)} issue${
+      totalIssues > 1 ? 's' : ''
+    }...`,
+  );
+
+  for (const category of issues) {
+    logger.log(`\n${chalk.dim(category.label)}`);
+
+    const issuesToFix = category.issues.filter(issue => issue.needsToBeFixed);
+
+    for (const issueToFix of issuesToFix) {
+      const issueSpinner = ora({
+        prefixText: '',
+        text: issueToFix.label,
+      }).start();
+
+      try {
+        await issueToFix.runAutomaticFix({loader: issueSpinner});
+      } catch (error) {
+        // TODO: log the error in a meaningful way
```

Comment:
Would console.error be better than nothing for now?

Reply:
Didn't pay much attention to this part yet, it's hard to log an error right now as the loader is on top of everything, so first it needs to stop the loader, log an error and then continue to the next automatic fix.